lib/yggdrasil/command/parser_rule.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Yggdrasil.Command.ParseRule do
```

Comment:
We might want to test this or least come back to it. It's relatively simple, so probably only new a few cases but helps build confidence in the functions the parser_builder is using.